codalab/worker_manager/gcp_batch_worker_manager.py

```diff
@@ -1,6 +1,13 @@
 try:
     from google.cloud.container_v1 import ClusterManagerClient  # type: ignore
     from google.oauth2 import service_account  # type: ignore
```

Comment:
Does running `pip install google-cloud-container` also install the required library for `google.oauth2`?

Reply:
yup it does